src/main/java/uk/gov/hmcts/darts/cache/token/service/value/impl/ServiceContextCacheValue.java

```diff
@@ -76,8 +76,8 @@ public ServiceContext getServiceContext() throws CacheException {
         return context;
     }
 
-    private String getUserName() throws CacheException {
-        ServiceContext serviceContext = getServiceContext();
+    private static String getUserName(ServiceContext context) throws CacheException {
+        ServiceContext serviceContext = context;
```

Comment:
I don't think we need these lines on 80 and 90 - maybe consider renaming the method param just